windows-observ-lib/panels.libsonnet

```diff
@@ -16,45 +16,45 @@ local utils = commonlib.utils;
             t.osInfo
             + g.query.prometheus.withFormat('table')
             + g.query.prometheus.withInstant(true)
-            + g.query.prometheus.withRefId('OS Info'),
+            + g.query.prometheus.withRefId('os info'),
```

Comment:
just curious why all these lowercase changes?

Reply:
Not sure how this happened. It is unintended.